sensor_msgs/Image image
geometry_msgs/TransformStamped pre_pose
geometry_msgs/TransformStamped post_pose